catalogue/webapp/pages/work.js

```diff
@@ -73,24 +73,21 @@ export const WorkPage = ({
   const licenseInfo =
     iiifImageLocationLicenseId && getLicenseInfo(iiifImageLocationLicenseId);
 
-  const sierraId = (
-    work.identifiers.find(
-      identifier => identifier.identifierType.id === 'sierra-system-number'
-    ) || {}
-  ).value;
+  const sierraIdFromPresentationManifestUrlMatch =
+    iiifPresentationLocation &&
+    iiifPresentationLocation.url &&
+    iiifPresentationLocation.url.match(/iiif\/(.*)\/manifest/);
 
-  const iiifPresentationLocation = (getIiifPresentationLocation(work) || {})
-    .url;
   const sierraIdFromPresentationManifestUrl =
-    iiifPresentationLocation &&
-    iiifPresentationLocation.match(/iiif\/(.*)\/manifest/)[1];
+    sierraIdFromPresentationManifestUrlMatch &&
+    sierraIdFromPresentationManifestUrlMatch[1];
 
   // We strip the last character as that's what Wellcome library expect
   const encoreLink =
-    sierraId &&
-    `http://search.wellcomelibrary.org/iii/encore/record/C__R${sierraId.substr(
+    sierraIdFromPresentationManifestUrl &&
```

Comment:
This isn't always the case, see the example [here](https://github.com/wellcometrust/wellcomecollection.org/pull/4232).

If we use the ID `b22021814` on [`v6zxz262`](https://wellcomecollection.org/works/v6zxz262) we would be using the wrong one.

Might be worth catching up with @pollecuttn to see what the logic we might be able to infer here, and maybe think about getting it into the API.

Reply:
Actually looking at the comment @pollecuttn made, we hold two versions in the library, the book, and the e-book, which are available in the library or online respectively.

Makes me think that, for now, we might keep linking to the one from the manifest, but bare this in mind, and potentially get this labelling in the API.

Reply:
Could we write a helper function in [`utils/works`](https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/utils/works.js)?

Reply:
> Makes me think that, for now, we might keep linking to the one from the manifest, but bare this in mind, and potentially get this labelling in the API.

After discussion with @jennpb @Heesoomoon @pollecuttn and @gestchild, we resolved to change this
![Screenshot 2019-03-12 at 10 50 47](https://user-images.githubusercontent.com/1394592/54194697-c3208080-44b4-11e9-87b4-5b9c2dde300a.png)

to something more like this
![Screenshot 2019-03-12 at 10 49 52](https://user-images.githubusercontent.com/1394592/54194708-c9aef800-44b4-11e9-9e74-adc0302205f4.png)

_only_ linking to physical versions.